.storybook/webpack.config.js

```diff
@@ -14,6 +14,6 @@ module.exports = function(config) {
         rules.less({ browsers }),
       ],
     },
-    plugins: [...config.plugins, plugins.extractText()],
+    // plugins: [...config.plugins, plugins.extractText()],
```

Comment:
Storybook cannot start on my local, it complains plugins.extractText is not a function, commented out temporarily